pkgs/by-name/gn/gnucash/package.nix

```diff
@@ -107,6 +130,7 @@ stdenv.mkDerivation rec {
 
   doCheck = true;
   enableParallelChecking = true;
+  enableParallelBuilding = true;
```

Comment:
That is already set by cmake, it shouldn't be repeated

Reply:
Ok!